plugins/BEdita/Core/tests/TestCase/Model/Action/ListRelatedFoldersActionTest.php

```diff
@@ -48,6 +48,7 @@ class ListRelatedFoldersActionTest extends TestCase
      * @return void
      *
      * @covers ::execute()
+     * @covers \BEdita\Core\Model\Action\ListRelatedObjectsAction::initialize()
```

Comment:
Uhm… we're not actually covering that method, testing its functionality or whatever… we're just "using" it. There's the [`@uses`](http://phpunit.readthedocs.io/en/7.1/annotations.html#uses) tag for these cases, but we're not using it anywhere because it would be too verbose.